shake/Build.hs

```diff
@@ -111,6 +111,7 @@ fixGhcPrimConf = do
 buildLibrary :: Bool -> String -> [String] -> Action ()
 buildLibrary debug lib deps = do
   let dir = library lib
+      externalDepLibs = ["ghc-boot"]
```

Comment:
Can you put this at the top level since it's constant? Easier to change later on.